adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/Platforms/netstandard1.1/PlatformParameters.cs

```diff
@@ -30,9 +30,15 @@
 
 namespace Microsoft.IdentityModel.Clients.ActiveDirectory
 {
+    /// <summary>
+    ///  Additional parameters used in acquiring user's authorization, specific to netstadard 1.1
```

Comment:
`netstandard 1.1` needs an "n"  here and in 1.3 file. Thanks for adding this (I should have done this when I did the webview changes)